index.html

```diff
@@ -195,90 +203,95 @@ <h2 class="works-card-title">Multi-Post Stories</h2>
         </li>
       </ul>
     </section>
+
     <section class="about">
-      <div class="about-heading">
-        <h2 class="about-about-h2">About Myself</h2>
-        <p class="about-desc">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <h3 class="about-conn">Let's Connect</h3>
-        <ul class="about-icons">
-          <li class="about-icons-li">
-            <a href="#" class="about-icons-li-a"
-              ><img src="/assets/github.svg.svg" alt="github icon"
-            /></a>
-          </li>
-          <li class="about-icons-li">
-            <a href="#" class="about-icons-li-a"
-              ><img src="/assets/twitter.svg.svg" alt="github icon"
-            /></a>
-          </li>
-          <li class="about-icons-li">
-            <a href="#" class="about-icons-li-a"
-              ><img src="/assets/angellist.svg.svg" alt="github icon"
-            /></a>
+      <div class="about-me-description">
+        <div class="about-heading">
+          <h2 class="about-about-h2">About Myself</h2>
+          <p class="about-desc">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h3 class="about-conn">Let's Connect</h3>
+          <ul class="about-icons">
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="/assets/github.svg.svg" alt="github icon"
+              /></a>
+            </li>
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
```

Comment:
- Good job designing the `About` section but it still has a few issues that need to be addressed 😄 
- [ ] Your icons are not displaying. Kindly check the icon names and paths and make sure they are displayed so that your site can have a better user interface.
- [ ] Your technology categories images are wrongly aligned. It would be great to use the right alignment so that your site can look more like the template.
- [ ] The `let's connect` heading has the wrong alignment also. It would be good to address that too.
- [ ] Also the `Get my resume` button is missing from your site. Kindly fix this so that your site can follow the design 👍 
- Your design:
![Annotation 2022-02-26 102252](https://user-images.githubusercontent.com/85113567/155838154-669a9c81-05a3-4425-a9f9-16fc1308ad74.jpg)
- Figma design:
![Annotation 2022-02-26 102310](https://user-images.githubusercontent.com/85113567/155838161-558c7b0f-f612-4048-b906-0971ac184549.jpg)

Reply:
* Fixed icons in About Myself